### PR TITLE
add text to pde_constrained.md

### PR DESCRIPTION
**Pull request by JeremyFongSP**

### CODE REVIEW

File: docs/src/examples/pde_constrained.md [hunk 1]
```diff
@@ -123,8 +127,29 @@ tspan    = (t0,tMax)
...
+
+`p` = true solution
+
+`xtrs` = convenient grouping of `dx` and `Nx` into Array
+
+`dt` = time distane between two points
+
+`t` (`t0` to `tMax`) spans the specified time frame
+
+`tspan` = span of `t`
```
Comment (ChrisRackauckas) [inline docs/src/examples/pde_constrained.md:146]:
```suggestion
In plain terms, the quantities that were defined are:

- `x` (to `Lx`) spans the specified 1D space
- `dx` = distance between two points
- `Nx` = total size of space
- `u0` = initial condition
- `p` = true solution
- `xtrs` = convenient grouping of `dx` and `Nx` into Array
- `dt` = time distane between two points
- `t` (`t0` to `tMax`) spans the specified time frame
- `tspan` = span of `t`
```

File: docs/src/examples/pde_constrained.md [hunk 2]
```diff
@@ -137,48 +162,66 @@ end
...
 # Testing Solver on linear PDE
 prob = ODEProblem(burgers,u0,tspan,p)
-sol = concrete_solve(prob,Tsit5(), dt=dt,saveat=t);
+sol = solve(prob,Tsit5(), dt=dt,saveat=t);
 
 plot(x, sol.u[1], lw=3, label="t0", size=(800,500))
 plot!(x, sol.u[end],lw=3, ls=:dash, label="tMax")
 ```
 
 ### Using Neural Network for Predictions
```
Comment (ChrisRackauckas) [inline docs/src/examples/pde_constrained.md:199]: Let's not call it a neural network. It's just a forward model for the PDE, or a prediction function.

File: docs/src/examples/pde_constrained.md [hunk 3]
```diff
@@ -221,13 +275,20 @@ plot!(PRED[end][:,end], lw=2, label="Prediction")
 
 ### Train
 
+The network is trained using `sciml_train` via backpropagation. The specifics of our
+1-layer NN is stored in `res` and `res.minimizer` returns the parameters that
+minimizes the cost function.
```
Comment (ChrisRackauckas) [inline docs/src/examples/pde_constrained.md:280]:
```suggestion
The parameters are trained using `sciml_train` and adjoint sensitivities. The resulting
best parameters are stored in `res` and `res.minimizer` returns the parameters that
minimizes the cost function.
```

File: docs/src/examples/pde_constrained.md [hunk 4]
```diff
@@ -213,6 +263,10 @@ cb(ps,loss(ps)...) # Testing callback function
 
 ### Plotting Prediction vs Ground Truth
 
+The scatter points plotted here are the ground truth obtained from the actual solution we
+solved for above. The solid line represents our prediction. The goal is for both to overlap
+almost perfectly when the NN finishes its training and the loss is close to 0.
```
Comment (ChrisRackauckas) [inline docs/src/examples/pde_constrained.md:268]:
```suggestion
almost perfectly when the PDE finishes its training and the loss is close to 0.
```

File: docs/src/examples/pde_constrained.md (hunk 2 above)
Comment (JeremyFongSP) [reply docs/src/examples/pde_constrained.md:193]: That's fair. Should I modify the 1-layer neural network comment as well?

Comment (ChrisRackauckas) [reply docs/src/examples/pde_constrained.md:193]: yes
